etna/lib/commands.rb

```diff
@@ -24,10 +24,12 @@ def dispatch_to_subcommand(cmd = 'help', *args, environment: nil, **kwds)
   def environment
     if @environment
       @environment
-    elsif @config && @config.keys.length == 1
+    elsif @config && @config.is_a?(Hash) && @config.keys.length == 1
       @config.keys.last.to_sym
+    elsif @config && @config.is_a?(Hash) && @config.keys.length > 1
+      raise "You have multiple environments configured, please specify your environment by adding --environment #{@config.keys.join("|")}"
     else
-      :production
+      raise "You do not have a successfully configured environment, please run #{program_name} config set https://polyphemus.ucsf.edu"
```

Comment:
Better feedback about when the user tries to run co,mands before an environment has been configured.